test/controllers/foreman/organizations_controller_test.rb

```diff
@@ -29,4 +29,44 @@ def test_use_newly_created_org
     assert_redirected_to step2_organization_path(:id => org.to_param)
     assert_equal session[:organization_id], org.id
   end
+
+  def test_simple_content_access_disable
+    # use org with SCA enabled
+    org = get_organization(:organization2)
+    Organization.any_instance.stubs(:simple_content_access?).returns true
+    # assert correct task will be started
+    @controller.expects(:async_task).with(::Actions::Katello::Organization::SimpleContentAccess::Disable, org.id.to_s)
+    # send request to disable SCA
+    put :update, params: { id: org.id, simple_content_access: false }
+  end
+
+  def test_simple_content_access_enable
+    # use org with SCA disabled
+    org = get_organization(:organization2)
+    Organization.any_instance.stubs(:simple_content_access?).returns false
+    # assert correct task will be started
+    @controller.expects(:async_task).with(::Actions::Katello::Organization::SimpleContentAccess::Enable, org.id.to_s)
+    # send request to enable SCA
+    put :update, params: { id: org.id, simple_content_access: true }
+  end
+
+  def test_simple_content_access_unchanged
+    # use org with SCA enabled
+    org = get_organization(:organization2)
+    Organization.any_instance.stubs(:simple_content_access?).returns true
+    # assert SCA task was not initiated
+    @controller.expects(:async_task).never
+    # update org and don't change SCA
+    put :update, params: { id: org.id, simple_content_access: true }
+  end
+
+  def test_edit_override_can_toggle
+    org = get_organization(:organization2)
+    Organization.any_instance.stubs(:service_level)
+    Organization.any_instance.stubs(:service_levels).returns []
+    Katello::UpstreamConnectionChecker.any_instance.expects(:can_connect?).returns true
+    Organization.any_instance.expects(:simple_content_access?).returns true
+    get :edit, params: { id: org.id }
```

Comment:
Mind adding the success response to these 4 tests? Sorry I missed that

Reply:
@jturel done, `update` actually responds with 302 Found so I asserted accordingly.